client/app/components/template-explorer/template-explorer.component.js

```diff
@@ -0,0 +1,204 @@
+/* eslint camelcase: "off" */
```

Comment:
Could you import the template here so that the component is more self-contained?

```javascript
import templateUrl from './template-explorer.html';
```

That way when you define the component you can use object shorthand and refer to `templateUrl` directly:

```javascript
  ...
  templateUrl,
};
```